src/components/Custom/HaveDesign/SelectKeyword.tsx

```diff
@@ -0,0 +1,145 @@
+    { value: '낙서 타투', checked: false },
+    { value: '라인 타투', checked: false },
+    { value: '레터링', checked: false },
+    { value: '캐릭터', checked: false },
+    { value: '일러스트', checked: false },
+    { value: '수채화', checked: false },
+  ];
+
+  const KEYWORDS_STYLE = [
+    { value: '귀여운', checked: false },
+    { value: '시크한', checked: false },
+    { value: '섬세한', checked: false },
+    { value: '심플한', checked: false },
+    { value: '키치한', checked: false },
+  ];
+
+  const [genreKeywords, setGenreKeywords] = useState(KEYWORDS_GENRE);
+  const [styleKeywords, setStyleKeywords] = useState(KEYWORDS_STYLE);
+
+  const handleKeywordChange = (index: number, type: string) => {
+    const checkedStyle = styleKeywords.filter((keyword) => keyword.checked).length;
+    const checkedGenre = genreKeywords.filter((keyword) => keyword.checked).length;
+    const checkedKeywords = checkedStyle + checkedGenre;
+    if (type === 'genre') {
+      const updatedKeywords = [...genreKeywords];
+      //전체 개수 제한
+      if (checkedKeywords >= 3 && updatedKeywords[index].checked === false) {
+        alert('스타일 최소 1개, 전체 최대 3개 선택 가능합니다.');
+      } else {
+        updatedKeywords[index].checked = !updatedKeywords[index].checked;
+        setGenreKeywords(updatedKeywords);
+      }
+    } else if (type === 'style') {
+      const updatedKeywords = [...styleKeywords];
+      //개수 제한
+      if (checkedKeywords >= 3 && updatedKeywords[index].checked === false) {
+        alert('스타일 최소 1개, 전체 최대 3개 선택 가능합니다.');
+      } else {
+        updatedKeywords[index].checked = !updatedKeywords[index].checked;
+        setStyleKeywords(updatedKeywords);
+      }
+    }
+  };
+  return (
+    <St.KeywordWrapper>
+      <St.Title>장르</St.Title>
+      <St.RadioWrapper>
+        {genreKeywords.map((keyword, index: number) => (
+          <St.RadioLabel key={index} htmlFor={keyword.value} checked={keyword.checked}>
+            <St.RadioInput
+              type='checkbox'
+              id={keyword.value}
+              name={keyword.value}
+              checked={keyword.checked}
+              onChange={() => handleKeywordChange(index, 'genre')}
+            />
+            <St.RadioText checked={keyword.checked}>
+              {keyword.checked ? <IcCheckSmallPink /> : null}
+              {keyword.value}
+            </St.RadioText>
+          </St.RadioLabel>
+        ))}
+      </St.RadioWrapper>
+      <St.Title>스타일</St.Title>
+      <St.RadioWrapper>
+        {styleKeywords.map((keyword, index: number) => (
+          <St.RadioLabel key={index} htmlFor={keyword.value} checked={keyword.checked}>
+            <St.RadioInput
+              type='checkbox'
+              id={keyword.value}
+              name={keyword.value}
+              checked={keyword.checked}
+              onChange={() => handleKeywordChange(index, 'style')}
+            />
+            <St.RadioText checked={keyword.checked}>
+              {keyword.checked ? <IcCheckSmallPink /> : null}
```

Comment:
요기 아이콘 영역을 null 값으로 주면 아이콘 빠졌을 때 스타일 잘 맞나용 ?? 궁금합니당!

Reply:
그리구 삼항연상자에서 null 값을 쓰는 것은 권장하지 않는다고 합니다!
```suggestion
              {keyword.checked && <IcCheckSmallPink /> }
```

이런 식으로 바꿔줄 수 있을것 같아요!